pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -270,7 +270,12 @@ public void initialize(final PulsarService pulsar) {
                             () -> LoadManagerShared.refreshBrokerToFailureDomainMap(pulsar, brokerToFailureDomainMap));
                 });
 
-        loadSheddingStrategy = createLoadSheddingStrategy();
+        if (placementStrategy instanceof LoadSheddingStrategy) {
+            // bind the load shedding strategy and the placement strategy
+            loadSheddingStrategy = (LoadSheddingStrategy) placementStrategy;
+        } else {
+            loadSheddingStrategy = createLoadSheddingStrategy();
```

Comment:
Can users set the `AvgShedder` for shedding strategy and use another `ModularLoadManagerStrategy`?

Reply:
No, that is a bad combination. There are already too many kind of combinations that user can choose, but only few of them can work well.

Reply:
So we should add a check for this case.

Reply:
If we use `AvgShedder` for `LoadSheddingStrategy`, and use others as `ModularLoadManagerStrategy`, we should throw out the exceptions? or should we log an error only?

Reply:
I think we can throw an exception in this case.

Reply:
Done, thanks.